compiler/optimizer/PartialRedundancy.cpp

```diff
@@ -230,6 +230,10 @@ int32_t TR_PartialRedundancy::perform()
    if (comp()->getProfilingMode() == JitProfiling && comp()->getHCRMode() != TR::none && _numProfilingsAllowed == 0)
       return 0;
 
+   // PRE is not supported for field watch yet
```

Comment:
I think this needs at least a reason - why isn't it supported, what is the TODO that needs to be addressed or the issue encountered?

Reply:
added reason and TODO.